gmplot/gmplot.py

```diff
@@ -1,22 +1,9 @@
-from __future__ import absolute_import
-
-import json
 import math
-import os
 import requests
-import warnings
-
-from collections import namedtuple
-
-from gmplot.color_dicts import mpl_color_map, html_color_codes
-from gmplot.google_maps_templates import SYMBOLS, CIRCLE
-
-
-Symbol = namedtuple('Symbol', ['symbol', 'lat', 'long', 'size'])
-
```

Comment:
Seems very similar to the changes made in #93.

@jurrutiag if you could address the same [comment](https://github.com/gmplot/gmplot/pull/93/files#r415562028) I left there, I'd be more than happy to review and merge the label and info window changes here.